### PR TITLE
Documentation update

### PR DESCRIPTION
**Pull request by markknol**

### CODE REVIEW

**Comment by Simn** [comment]
I'll go through that tomorrow, at a first glance some things sound a bit weird (e.g. the description of the process pipes).

**Comment by Simn** [comment]
`/home/travis/build/HaxeFoundation/haxe/std/cpp/Lib.hx:68: characters 1-7 : Unexpected public`

File: std/sys/io/Process.hx [hunk 1]
```diff
@@ -22,10 +22,20 @@
 package sys.io;
 
 extern class Process {
+	/**
+		Standard output. The output stream where a process writes its output data.
+	**/
+	var stdout(default, null) : haxe.io.Input;
+
+	/**
+		Standard error. The output stream to output error messages or diagnostics.
```
Comment (Simn) [inline std/sys/io/Process.hx]: This requires at least a "typically", but I'm not quite sure if we should add these layman-one-liners at all.

Comment (markknol) [inline std/sys/io/Process.hx]: I don't know what a layman-one-liners is.

**Pull request merged**